CHANGELOG.md

```diff
@@ -2,6 +2,29 @@
 
 All notable changes to this project will be documented in this file.
 
+## [v1.3.0](https://github.com/voxpupuli/container-puppetdb/tree/v1.3.0) (2024-01-31)
+
+[Full Changelog](https://github.com/voxpupuli/container-puppetdb/compare/v1.2.1...v1.3.0)
+
+**Implemented enhancements:**
+
+- PuppetDB 7.16.0 and 8.3.0 [\#44](https://github.com/voxpupuli/container-puppetdb/issues/44)
+- Update puppetdb packages [\#45](https://github.com/voxpupuli/container-puppetdb/pull/45) ([tuxmea](https://github.com/tuxmea))
+
+**Closed issues:**
+
+- Add support for custom initialization scripts [\#39](https://github.com/voxpupuli/container-puppetdb/issues/39)
+- Implement shellcheck in CI [\#34](https://github.com/voxpupuli/container-puppetdb/issues/34)
```

Comment:
Iadded the skip-changelog label because we've a related PR already in the changelog.